.travis.yml

```diff
@@ -18,12 +17,12 @@ matrix:
   include:
     - php: 5.3
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
What's the reason for removing PHP 5.3 and 5.4 from travis? versions 2.* need to work on PHP 5.3